proxy-node-acceptance-tests/features/smoke/production/netherlands.feature

```diff
@@ -0,0 +1,6 @@
+Feature: eIDAS Proxy Node Smoke Test - Netherlands - Production
+
+    Scenario: Proxy Node Netherlands happy path - LOA Substantial
+        Given   the user visits the "Netherlands" Stub Connector Node page
+        And     they navigate the "Netherlands" journey to verify with UK identity
+        Then    they should arrive at the Verify Hub start page
```

Comment:
No newline at the end of file could this be an issue?

Reply:
I've run the tests locally, Cucumber doesn't complain and the tests pass, so I think it's fine